public/index.html

```diff
@@ -27,7 +27,14 @@ <h2 class="text-2xl text-green-50 italic">Frontend Developer-Writer</h2>
         <div class="absolute top-0 right-0 mt-1 mr-1">
             <img src="images/2202-2020-101115569136863779874-01 (1)-01-01.jpeg.jpg" alt="Adhithya Rajan's Photo" class="max-w-md rounded-full shadow-lg">
         </div>
-        <main class="bg-zinc-900 text-pink-700">
+        <main class="bg-zinc-900 text-pink-700 relative flex">
+            <div class="w-1/4 p-40">
+                <div class="mt-1 mr-1">
+                    <a href="docs/adhithya-resume.pdf">
```

Comment:
open the resume in a new tab by adding the `target="_blank"` property.

Refer: https://www.w3schools.com/tags/att_a_target.asp